gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/control/InternalResourceManager.java

```diff
@@ -527,6 +527,15 @@ public LoadProbe getLoadProbe() {
   }
 
   /**
+   * This method is test purposes only.
+   */
+  public LoadProbe setLoadProbe(LoadProbe probe) {
+    LoadProbe old = this.loadProbe;
+    this.loadProbe = probe;
+    return old;
+  }
+
```

Comment:
perhaps better to have this as package private so only relevant tests (either in the same package or via a test helper class in the package) can use